Documentation/proposals/202405-scrapeconfig-graduation.md

```diff
@@ -0,0 +1,130 @@
+# Extending The Surface of The ScrapeConfig CRD
```

Comment:
should we rather advertise as "Graduate ScrapeConfig CRD to beta"?

Reply:
I agree, that seems more fitting.